tests/basic.html

```diff
@@ -43,11 +43,11 @@
         // add the catch to ensure you do not receive a "uncaught (in promise) warning"
         .catch(
           (err) => {
-            console.error(err)
+            console.error(err.message, err.cause)
           }
         );
 
-      butter.content.cancelRequest()
+      // butter.content.cancelRequest()
```

Comment:
Is this line and the one below (87) commented on purpose?

Reply:
nope. just commented out so I could test.